db/models.py

```diff
@@ -17,14 +23,19 @@ def __str__(self) -> str:
 
 
 class Movie(models.Model):
-    title = models.CharField(max_length=255)
+    title = models.CharField(max_length=255, db_index=True)
```

Comment:
either db_index = True or indexes should be used. indexes are often used for complex indexes